client/app/bundles/Attendance/components/ClassesScreen/ClassesList/ClassesList.jsx

```diff
@@ -13,8 +13,8 @@ export default class ClassesList extends BaseComponent {
   }
 
   render() {
-    const { $$classes } = this.props
-    const classes = $$classes && $$classes.get('classes') ? $$classes.get('classes').valueSeq() : [];
+    const { $$attendance } = this.props
```

Comment:
Would it not be better to have a list of classes as props instead of all available data?

Reply:
I'm not sure yet how much we want to narrow this down. If we wanted to display some information about who attended a class in the classes list, we'd need access to the students as well. I guess I could narrow it down to what's needed for now, and then expand it later.

Reply:
I see what you mean. It just kind of feels like a global object you know? Every component will need data, so all of them would have that one 'attendance' prop. Doesn't really feel like a clean interface. I don't know much about React and how it handles data - what was the reason for merging the data stores again?